admin/app/com/lucidchart/piezo/admin/controllers/HealthCheckController.scala

```diff
@@ -0,0 +1,27 @@
+package com.lucidchart.piezo.admin.controllers
+
+import org.joda.time.{DateTime,Minutes}
+import org.joda.time.format.ISODateTimeFormat
+import play.api._
+import play.api.Play.{current,configuration}
+import play.api.mvc._
+import scala.io.Source
+
+object HealthCheckController extends Controller {
+ def main = Action { implicit requests =>
+   if(workersAreHealthy) {
+     Ok
+   } else {
+     ServiceUnavailable
+   }
+
+ }
+
+ def workersAreHealthy(): Boolean = {
+   val workerTimestamp = (Source.fromFile(configuration.getString("com.lucidchart.piezo.heartbeatFile").get).getLines.toList)(0)
```

Comment:
also, if the file doesn't exist, I assume we want to return a 503, but I imagine this will throw an exception and return a 500